packages/fetch/src/cache.ts

```diff
@@ -1,4 +1,5 @@
 import { DEV } from "solid-js";
+import { urlToHttpOptions } from "url";
```

Comment:
Is it fine to import from "url" on the browser?

Reply:
Removed the stray auto-import.